### PR TITLE
[PATCH v7] linux-dpdk: pktio: add additional eal args using config file

### PR DESCRIPTION
**Pull request by vijayinavolu**
Adds a new function to get a string from config

Signed-off-by: Vijay Ram Inavolu <vinavolu@marvell.com>

### CODE REVIEW

File: config/odp-linux-dpdk.conf [hunk 1]
```diff
@@ -58,6 +58,9 @@ dpdk: {
 
 	# List of PCI devices that should not be used.
 	pci_blacklist = []
+
+	# Additional EAL arguments can be passed as a string
+	additional_eal_args = ""
```
[the remark this answers is not in the document]
Comment (MatiasElo) [reply config/odp-linux-dpdk.conf]: Ah, you are correct. I'd like to clarify the processing order of options and that `process_mode_memory_mb` uses `-m` EAL option. How about this?
```
# String in EAL parameters format. Parameters given here will be used as
# last arguments to rte_eal_init() after other DPDK options
# ('process_mode_memory_mb' (-m), 'pci_whitelist' (-w), 'pci_blacklist' (-b))
# and ODP_PLATFORM_PARAMS environment variable contents.
```

[the remark this answers is not in the document]
Comment (vijayinavolu) [reply config/odp-linux-dpdk.conf]: Yeah this is fine, updated it

File: config/odp-linux-dpdk.conf [hunk 2]
```diff
@@ -58,6 +58,12 @@ dpdk: {
 
 	# List of PCI devices that should not be used.
 	pci_blacklist = []
+
+	# String in EAL parameters format. Parameters given here will be used as
+	# last arguments to rte_eal_init() after other DPDK options
+	# ('process_mode_memory_mb' (-m), 'pci_whitelist' (-w), 'pci_blacklist' (-b))
+	# and ODP_PLATFORM_PARAMS environment variable contents.
```
Comment (MatiasElo) [inline config/odp-linux-dpdk.conf:65]: The short description was missing from my previous comment, sorry. Please change the text once more to:
```
	# EAL parameters string
	#
	# String is in EAL parameters format. Parameters given here will be used
	# as last arguments to rte_eal_init() after other DPDK options
	# ('process_mode_memory_mb' (-m), 'pci_whitelist' (-w), 'pci_blacklist'
	# (-b)) and ODP_PLATFORM_PARAMS environment variable contents.
```

I'll merge the PR after this final change is done.

Comment (vijayinavolu) [reply config/odp-linux-dpdk.conf]: Updated the comment

**Review by MatiasElo — APPROVED** [review]

**Pull request merged**